frontend/app/services/GuardianContentService.scala

```diff
@@ -123,6 +130,20 @@ trait GuardianContent {
     }
   }
 
+  def membershipFrontContentQuery(page: Int): Future[ItemResponse] = {
+    val itemQuery = ItemQuery("/membership")
+      .showFields("all")
```

Comment:
Do we need all in the case? Just thinking the responses will be fairly large, we could cut this down by just requesting the fields we need?